src/main/java/core/basesyntax/Figures.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum Figures {
+    SQUARE,
+    RECTANGLE,
+    RIGHT_TRIANGLE,
+    CIRCLE,
+    ISOSCELES_TRAPEZOID,
+}
```

Comment:
Do you actually need this enum?

What if we will create another `Figure` subclass? Will we always update this enum accordingly?

Let's try not to use it. See comments tips in `FigureSupplier`.

Reply:
done